honeybee_grasshopper_core/src/HB Apertures by Ratio.py

```diff
@@ -56,29 +56,16 @@
             here, different operable properties will be assigned based on
             cardinal direction, starting with north and moving clockwise.
             Default: False.
-        ep_constr_: Optional text for an energy construction to be used for all
-            generated apertures. This text will be used to look up a construction
-            in the window construction library. This can also be a custom
-            WindowConstruction object. If an array of text or construction objects
-            are input here, different constructions will be assigned based on
-            cardinal direction, starting with north and moving clockwise.
-        rad_mat_: Optional text for a radiance material to be used for all
-            generated apertures. This text will be used to look up a material
-            in the material library. This can also be a custom material object.
-            If an array of text or material objects are input here, different
-            materials will be assigned based on cardinal direction, starting
-            with north and moving clockwise.
-        _run: Set to True to run the component.
```

Comment:
Is it by accident or you have removed run from this component?

Reply:
That was by accident. Good catch!